cmd/serve/dlna/dlna.go

```diff
@@ -97,12 +97,25 @@ func newServer(f fs.Fs, opt *dlnaflags.Options) *server {
 	if friendlyName == "" {
 		friendlyName = makeDefaultFriendlyName()
 	}
+	interfaces := make([]net.Interface, 0, len(opt.InterfaceNames))
+	for _, interfaceName := range opt.InterfaceNames {
+		var err error
+		tmp, err := net.InterfaceByName(interfaceName)
+		if err != nil {
+			fs.Errorf(nil, "Failed to resolve interface name: %v", err)
+			continue
```

Comment:
Should we return here instead, i.e. if the user provided configuration is wrong? (Also if the user-provided interface(s) do(es) not contain the correct flags.)